README.md

```diff
@@ -1,102 +1,90 @@
+`iron-selector` is an element which can be used to manage a list of elements
+that can be selected. Tapping on the item will make the item selected. The
+`selected` indicates which item is being selected. The default is to use the
+index of the item.
 
-  Example:
+See: [Documentation](https://www.webcomponents.org/element/@polymer/iron-selector),
+  [Demo](https://www.webcomponents.org/element/@polymer/iron-selector/demo/demo/index.html).
 
-```html
-  <iron-selector selected="0">
-    <div>Item 1</div>
-    <div>Item 2</div>
-    <div>Item 3</div>
-  </iron-selector>
-```
+## Usage
 
-  If you want to use the attribute value of an element for `selected` instead of the index,
-  set `attrForSelected` to the name of the attribute.  For example, if you want to select item by
-  `name`, set `attrForSelected` to `name`.
+### Installation
 
-  Example:
-
-```html
-  <iron-selector attr-for-selected="name" selected="foo">
-    <div name="foo">Foo</div>
-    <div name="bar">Bar</div>
-    <div name="zot">Zot</div>
-  </iron-selector>
+```
+npm install --save @polymer/iron-selector
 ```
 
-  You can specify a default fallback with `fallbackSelection` in case the `selected` attribute does
-  not match the `attrForSelected` attribute of any elements.
-
-  Example:
+### In an HTML file
 
 ```html
-    <iron-selector attr-for-selected="name" selected="non-existing"
-                   fallback-selection="default">
-      <div name="foo">Foo</div>
-      <div name="bar">Bar</div>
-      <div name="default">Default</div>
+<html>
+  <head>
+    <script type="module">
+      import '@polymer/iron-selector/iron-selector.js';
+    </script>
+  </head>
+  <body>
+    <iron-selector selected="0">
+      <div>Item 1</div>
+      <div>Item 2</div>
+      <div>Item 3</div>
     </iron-selector>
+  </body>
+</html>
 ```
 
-  Note: When the selector is multi, the selection will set to `fallbackSelection` iff
-  the number of matching elements is zero.
+### In a Polymer 3 element
 
-  `iron-selector` is not styled. Use the `iron-selected` CSS class to style the selected element.
+```js
+import {PolymerElement} from '@polymer/polymer/polymer-element.js';
+import {html} from '@polymer/polymer/lib/utils/html-tag.js';
 
-  Example:
+import '@polymer/iron-selector/iron-selector.js';
 
-```html
-  <style>
-    .iron-selected {
-      background: #eee;
-    }
-  </style>
-
-  ...
-
-  <iron-selector selected="0">
-    <div>Item 1</div>
-    <div>Item 2</div>
-    <div>Item 3</div>
-  </iron-selector>
+class ExampleElement extends PolymerElement {
+  static get template() {
+    return html`
+      <iron-selector selected="0">
+        <div>Item 1</div>
+        <div>Item 2</div>
+        <div>Item 3</div>
+      </iron-selector>
+    `;
+  }
+}
+
+customElements.define('example-element', ExampleElement);
 ```
 
-### Notable breaking changes between 1.x and 2.x (hybrid):
+## Contributing
 
-#### IronSelectableBehavior
```

Comment:
I would maybe list these in the small summary blurb as well. Like, iron-selector is ... This package also provides two behaviours, x and y, for  implementing your own selector (or whatever)

Reply:
I added more info on `IronSelectableBehavior` and `IronMultiSelectableBehavior` - is that what you were looking for?